frontend/src/views/instructors/editable-instructors-list.js

```diff
@@ -42,7 +58,38 @@ function EditableInstructorsList(props) {
         );
     }
 
+    const instructorCurrentlyAssignedHash = {};
+    (positions || []).map((position) =>
+        position.instructors.map(
+            (instructor) =>
+                (instructorCurrentlyAssignedHash[instructor.id] = true)
+        )
+    );
+
     const columns = [
+        {
+            Header: (
+                <FaTrash className="delete-instructor-column-header-icon" />
+            ),
+            Cell: (props) => {
+                // Renders a custom cell with a delete button if the instructor is not currently assigned to any position, otherwise nothing
```

Comment:
This should be a docstring. It would be better to split this out as its own function component.